src/index.html

```diff
@@ -3,10 +3,10 @@
 
 <head>
   <meta charset="utf-8">
-  <title>Patternfly Seed</title>
+  <title>OCP Metering</title>
   <meta id="appName" name="application-name" content="Patternfly Seed">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="icon" type="image/png" href="https://www.patternfly.org/components/patternfly/dist/img/favicon.ico">
+  <link rel="icon" type="image/png" sizes="32x32" href="https://raw.githubusercontent.com/skanthed/ssmt/master/favicon.ico">
```

Comment:
Please use this:

`<link rel="icon" type="image/png" href="https://massopen.cloud/wp-content/uploads/2020/02/cropped-moc-4-32x32.png">`